dist-persist/wbstack/src/Shim/Web.php

```diff
@@ -7,13 +7,14 @@
     \WBStack\Info\GlobalSet::forDomain($_SERVER['SERVER_NAME']);
 } catch (\WBStack\Info\GlobalSetException $ex) {
     http_response_code($ex->getCode());
-    echo "You have requested the domain: " . $_SERVER['SERVER_NAME'] . ". But that wiki can not currently be loaded.".PHP_EOL;
+    echo "You have requested the domain: " . $_SERVER['SERVER_NAME'] . ". But that wiki can not currently be loaded.<br/>";
```

Comment:
Replacing `PHP_EOL` with `<br>` is a drive by fix as I noticed this currently did not work as intended.